server/scripts/update-operators.sh

```diff
@@ -1,10 +1,17 @@
 #!/bin/sh
+
```

Comment:
Can we `pushd` up one level and to `data` or something? Having the `community-operators` directory under scripts doesn't seem reasonable.

Reply:
That makes much more sense - I'll add that now